### PR TITLE
Plt 2045

### PR DESCRIPTION
**Pull request by vertigoc**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ## Release History
 
 - Unreleased
```
Comment (eliasem) [inline CHANGELOG.md]: Remove this line

File: bower.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "simcapi-js",
```
Comment (eliasem) [inline bower.json]: Can you do me a favour and set the name to `simcapi` to match the package.

**Comment by eliasem** [comment]
Just those small changes please.

**Comment by vertigoc** [comment]
:chicken: :chicken: :chicken: :chicken: :chicken: :chicken: :chicken: :chicken: :chicken: :chicken:

**Comment by jennifermckenzie** [comment]
👍

**Pull request merged**